main.py

```diff
@@ -0,0 +1,48 @@
+from fastapi import FastAPI, Depends
+from fastapi_crudrouter import SQLAlchemyCRUDRouter
```

Comment:
Do you really need `fastapi_crudrouter` for the educational project? Please, rewrite your code without `fastapi_crudrouter`